src/Cake.Common/Tools/DotNetCore/Build/DotNetCoreBuildSettings.cs

```diff
@@ -42,11 +37,6 @@ public sealed class DotNetCoreBuildSettings : DotNetCoreSettings
         public string VersionSuffix { get; set; }
```

Comment:
I was planning to add `VersionPrefix` after all the current tasks are done, along with `Properties`. I don't think we should use `Version`, I think it gets overridden in some situations unexpectedly and doesn't work with `VersionSuffix`.  

@patriksvensson if my memory is correct, when I was testing using `VersionPrefix` with the build command acted as `AssemblyVersion`